### PR TITLE
Footprints

### PR DESCRIPTION
**Pull request by marySalvi**
Introduced Functionality:
- Depicts footprints of results
- Removes need for Cesium Token
- Adds open source base layer maps
- Makes CesiumViewer and SearchBar full screen
- Intersects default predicate

Aware that this need refactoring, specifically removing business logic from cesium component. Merging should be independent of that so we can move away from the quasi-dev state this branch has adapted.

### CODE REVIEW

**Review by mvandenburgh — APPROVED** [review]
I agree that some refactoring should happen, but for now this LGTM

File: src/main.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
-import axios from 'axios';
-import OauthClient from '@girder/oauth-client';
 import * as Sentry from '@sentry/vue';
 import Vue from 'vue';
 import '@/plugins/composition';
+// import DatetimePicker from 'vuetify-datetime-picker';
```
Comment (annehaley) [inline src/main.ts:4]: Do we need this commented here?

**Review by annehaley — APPROVED** [review]
The portion about `imageryViewModels` in `CesiumViewer.vue` is cool :+1: Thanks for letting me review

File: src/main.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 import Cesium from './plugins/cesium';
 import App from './App.vue';
 import router from './router';
 import vuetify from './plugins/vuetify';
 import { restoreLogin, oauthClient, axiosInstance } from './api/rest';
 
+// Set token to `null` to avoid warning
 Cesium.Ion.defaultAccessToken = process.env.VUE_APP_CESIUM_ACCESS_TOKEN;
```
Comment (mvandenburgh) [inline src/main.ts:12]:
```suggestion
Cesium.Ion.defaultAccessToken = process.env.VUE_APP_CESIUM_ACCESS_TOKEN;
```
```suggestion
Cesium.Ion.defaultAccessToken = null;
```

Comment (marySalvi) [reply src/main.ts:12]: I was originally thinking of keeping this and then updating the readme but on second thought it probably is better to change it here.

File: src/main.ts (hunk 1 above)
Comment (marySalvi) [reply src/main.ts:4]: No we don't, good catch.

**Pull request merged**